spec/suma/admin_api/members_spec.rb

```diff
@@ -234,4 +234,25 @@ def make_item(i)
       expect(last_response).to have_status(403)
     end
   end
+
+  describe "DELETE /v1/members/:id/memberships" do
```

Comment:
Let's figure out how we can bulk create/update/delete memberships rather than the endpoints here.